support-frontend/assets/helpers/abTests/lpPreviousGiving.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { getQueryParameter } from 'helpers/urls/url';
+
+// ----- Consts ----- //
+
+const SUPPORT_AGAIN_HEADER_TEST_NAME = 'header-support-again';
+const SUPPORT_AGAIN_HEADER_VARIANT_NAME = 'control';
+
+// ----- Helpers ----- //
+
+export function getLongMonth(date: Date) {
+  return date.toLocaleDateString('default', { month: 'long' });
+}
+
+function nth(d) {
+  if (d >= 11 && d <= 13) {
+    return 'th';
+  }
+  switch (d % 10) {
+    case 1:
+      return 'st';
+    case 2:
+      return 'nd';
+    case 3:
+      return 'rd';
+    default:
+      return 'th';
+  }
+}
+
+export function getDateWithOrdinal(date: Date) {
+  const dayOfMonth = date.getDate();
+
+  return `${dayOfMonth}${nth(dayOfMonth)}`;
+}
+
+export function isInSupportAgainHeaderVariant() {
+  const acquisitionData = JSON.parse(decodeURI(getQueryParameter('acquisitionData') || '{}'));
```

Comment:
Rather than parsing the querystring again can we pass the `referrerAcquisitionData` in from `ContributionFormContainer`?

Reply:
Ahh yeah ofc! I'll make that change now